### PR TITLE
add variable item `tidb_track_aggregate_memory_usage`

### PR DESCRIPTION
**Pull request by wshwsh12**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)
Introduce variable `tidb_track_aggregate_memory_usage` by https://github.com/pingcap/tidb/pull/21597

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: system-variables.md [hunk 1]
```diff
@@ -995,3 +995,10 @@ explain select * from t where age=5;
 - 默认值：0.8
 - TiDB 内存使用占总内存的比例超过一定阈值时会报警。该功能的详细介绍和使用方法可以参考 [`memory-usage-alarm-ratio`](/tidb-configuration-file.md#memory-usage-alarm-ratio-从-v409-版本开始引入)。
 - 该变量的初始值可通过 [`memory-usage-alarm-ratio`](/tidb-configuration-file.md#memory-usage-alarm-ratio-从-v409-版本开始引入) 进行配置。
+
+### `tidb_track_aggregate_memory_usage`
+
+- 作用域：SESSION | GLOBAL
+- 默认值：OFF
+- 这个变量表示是否跟踪聚合函数的内存使用情况。
```
Comment (XuHuaiyu) [inline system-variables.md:1003]: 跟踪 --> 追踪

Comment (XuHuaiyu) [inline system-variables.md:1003]: - 这里需要再写清楚当开启该功能时，聚合函数内存会被统计，进而可能会造成整个 sql 内存统计值超阈值，然后被 OOM-ACTION 定义的行为影响
- 另外需要协商，这个功能是从 5.0-rc 版本才开始提供

File: system-variables.md [hunk 2]
```diff
@@ -995,3 +995,10 @@ explain select * from t where age=5;
 - 默认值：0.8
 - TiDB 内存使用占总内存的比例超过一定阈值时会报警。该功能的详细介绍和使用方法可以参考 [`memory-usage-alarm-ratio`](/tidb-configuration-file.md#memory-usage-alarm-ratio-从-v409-版本开始引入)。
 - 该变量的初始值可通过 [`memory-usage-alarm-ratio`](/tidb-configuration-file.md#memory-usage-alarm-ratio-从-v409-版本开始引入) 进行配置。
+
+### `tidb_track_aggregate_memory_usage` <span class="version-mark">从 v5.0-rc 版本开始引入</span>
+
+- 作用域：SESSION | GLOBAL
+- 默认值：OFF
+- 这个变量表示是否追踪聚合函数的内存使用情况。当开启该功能时，聚合函数内存会被统计，进而可能会造成整个 sql 内存统计值超阈值 [`mem-quota-query`](/tidb-configuration-file.md#mem-quota-query)，然后被 [`oom-action`](/tidb-configuration-file.md#oom-action) 定义的行为影响。
```
Comment (TomShawn) [inline system-variables.md:1003]:
```suggestion
- 这个变量表示是否追踪聚合函数的内存使用情况。当开启该功能时，聚合函数的内存使用情况会被统计，进而可能会造成整个 SQL 内存统计值超阈值 [`mem-quota-query`](/tidb-configuration-file.md#mem-quota-query)，然后被 [`oom-action`](/tidb-configuration-file.md#oom-action) 定义的行为影响。
```

File: system-variables.md [hunk 3]
```diff
@@ -995,3 +995,10 @@ explain select * from t where age=5;
...
+- **警告： `tidb_track_aggregate_memory_usage` 目前为实验性特性，不建议在生产环境中使用。**
```
Comment (TomShawn) [inline system-variables.md:1004]:
```suggestion
```

File: system-variables.md [hunk 4]
```diff
@@ -995,3 +995,10 @@ explain select * from t where age=5;
 - 默认值：0.8
 - TiDB 内存使用占总内存的比例超过一定阈值时会报警。该功能的详细介绍和使用方法可以参考 [`memory-usage-alarm-ratio`](/tidb-configuration-file.md#memory-usage-alarm-ratio-从-v409-版本开始引入)。
 - 该变量的初始值可通过 [`memory-usage-alarm-ratio`](/tidb-configuration-file.md#memory-usage-alarm-ratio-从-v409-版本开始引入) 进行配置。
+
+### `tidb_track_aggregate_memory_usage` <span class="version-mark">从 v5.0-rc 版本开始引入</span>
+
```
Comment (TomShawn) [inline system-variables.md:1000]:
```suggestion

> **警告：**
>
> `tidb_track_aggregate_memory_usage` 目前为实验特性，不建议在生产环境中使用。

```

File: system-variables.md [hunk 5]
```diff
@@ -995,3 +995,13 @@ explain select * from t where age=5;
 - 默认值：0.8
 - TiDB 内存使用占总内存的比例超过一定阈值时会报警。该功能的详细介绍和使用方法可以参考 [`memory-usage-alarm-ratio`](/tidb-configuration-file.md#memory-usage-alarm-ratio-从-v409-版本开始引入)。
 - 该变量的初始值可通过 [`memory-usage-alarm-ratio`](/tidb-configuration-file.md#memory-usage-alarm-ratio-从-v409-版本开始引入) 进行配置。
+
+### `tidb_track_aggregate_memory_usage` <span class="version-mark">从 v5.0-rc 版本开始引入</span>
```
Comment (TomShawn) [inline system-variables.md:999]:
```suggestion
### `tidb_track_aggregate_memory_usage` <!-- 从 v5.0-rc 版本开始引入 -->
```

**Review by TomShawn — APPROVED** [review]
LGTM
@XuHuaiyu PTAL again

**Review by XuHuaiyu — COMMENTED** [review]
LGTM

File: system-variables.md [hunk 6]
```diff
@@ -995,3 +995,13 @@ explain select * from t where age=5;
 - 默认值：0.8
 - TiDB 内存使用占总内存的比例超过一定阈值时会报警。该功能的详细介绍和使用方法可以参考 [`memory-usage-alarm-ratio`](/tidb-configuration-file.md#memory-usage-alarm-ratio-从-v409-版本开始引入)。
 - 该变量的初始值可通过 [`memory-usage-alarm-ratio`](/tidb-configuration-file.md#memory-usage-alarm-ratio-从-v409-版本开始引入) 进行配置。
+
+### `tidb_track_aggregate_memory_usage` <!-- 从 v5.0-rc 版本开始引入 -->
```
Comment (TomShawn) [inline system-variables.md:999]:
```suggestion
### `tidb_track_aggregate_memory_usage` <!-- 从 v5.0.0-rc 版本开始引入 -->
```

**Pull request merged**